conboarding.md

```diff
@@ -78,25 +82,31 @@ How much does it cost to run things?
       $SOME_PORT`
       Note: When logging in from campus (use `eduroam`), SSH does not always work on
       `typhon`. Instead it is recommended to use `ssh-agent` and forward the authentication connection using `-A`
-         1. (Assuming `ssh-agent` is running) `ssh-add -t 3600 /path/to/key`
+         1. (Assuming `ssh-agent` is running) `ssh-add -t 36000 /path/to/key`
```

Comment:
I tend to get kicked off if this value is set too low, but I guess it's just an 'opinion'